src/ASM/ASMs2Dmx.C

```diff
@@ -206,7 +206,7 @@ bool ASMs2Dmx::generateFEMTopology ()
         ASMmxBase::Type == ASMmxBase::DIV_COMPATIBLE)
       projB = proj = ASMmxBase::raiseBasis(surf);
     else
-      projB = proj = m_basis[2-ASMmxBase::geoBasis]->clone();
+      projB = proj = m_basis[0]->clone();
```

Comment:
or `.front()` (instead of `[0]`), like it was before d8d4d40d8ff9435a6971779913b6431135f568b7. So that fix (from Oct 2021) is no longer valid since reverted here? Or are there other cases when it still apply.

Reply:
the fix was partially broken, as in it fixes what is was supposed to fix for FRTH but it broke subgrid in the process. and now i almost broke FRTH while fixing subgrid.. too focused on problems at hand, not enough on the world